Assets/00_Content/Scripts/Scenes/SceneLoadManager.cs

```diff
@@ -45,18 +53,21 @@ public class SceneLoadManager : SingletonBehaviour<SceneLoadManager> {
 			}
 		}
 
-		public void LoadLobby() => Load(lobby);
-		public void LoadMainMenu() => Load(mainMenu);
-		public void LoadGame() => Load(game);
+		public void LoadLobby() => Load(lobby, false);
+		public void LoadMainMenu() => Load(mainMenu, true);
+		public void LoadGame() => Load(game, true);
 
-		private static void Load(SceneInfo sceneInfo) {
-			// Make sure player are carried over
+		private void Load(SceneInfo sceneInfo, bool doTransition) {
+			// Make sure players are carried over
 			if (PlayerManager.Instance != null) {
```

Comment:
I have noticed that you do this in the code from time to time. I'm not sure about what the purpose of this is.
Is it meant to ensure that you can run the code even if it won't be able to find the PlayerManager? Or is there something deeper here that I'm not understanding?

Reply:
Exactly, you are correct. These checks are there to enable the script to continue even if a certain gameobject is missing. You will just lose some functionality instead of the rest of the script/update for that frame.